development/docker-compose-observability.yml

```diff
@@ -69,9 +89,41 @@ services:
       - '--web.console.templates=/etc/prometheus/consoles'
       - '--storage.tsdb.retention.time=1d'
       - '--web.enable-lifecycle'
+      - '--web.enable-remote-write-receiver'
     ports:
       - "9090:9090"
 
+  node-exporter:
+    profiles: [debug]
+    image: prom/node-exporter:latest
+    restart: unless-stopped
```

Comment:
I think we should remove `restart: unless-stopped` for the dev environment, most container don't have it